set.mm

```diff
@@ -424917,8 +424922,9 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        <li> vv : theorem with two (main) disjoint variable conditions</li>
        <li> w : weak(er) form of a theorem</li>
        <li> ALT : alternate proof of a theorem</li>
-       <li> ALTV : alternate version of a theorem or definition</li>
-       <li> OLD : old/obsolete version of a theorem/definition/proof</li>
+       <li> ALTV : alternate version of a theorem or definition (mathbox
+       only)</li>
+       <li> OLD : old/obsolete version of a theorem or definition</li>
```

Comment:
Why did you remove "proof"? Often, or mostly, only the proof is different.

Reply:
I will restore it.

I had removed it because I was thinking of the type of statement: I intuitively translated "either a $p-statement or an $a-statement" to "either a theorem or a definition".  That is, a theorem always goes with a proof in metamath databases, so it's a bit strange to say "a theorem or a proof", but in the current situation it makes sense informally: it means "only the proof part of the $p-statement is different".

Reply:
This is still not restored!?

Reply:
Fine now.